include/klee/Core/Interpreter.h

```diff
@@ -57,7 +57,7 @@ class InterpreterHandler {
   virtual void incPathsCompleted() = 0;
   virtual void incPathsExplored(std::uint32_t num = 1) = 0;
 
-  virtual void processTestCase(const ExecutionState &state, const char *message,
+  virtual void processTestCase(ExecutionState &state, const char *message,
```

Comment:
Try to keep the state constant, it does not always terminated after the test is generated and can be used further for execution.

Reply:
Ok, then I should copy `state.symbolics` to another vector and carry it everywhere